### DISCUSSION TITLE
Cachet 3.x Design Implementation

### DISCUSSION THREAD

**Discussion by jbrooksuk** [cachethq/cachet#4355] [Announcements]
Hey everyone,

We're on the verge of being able to commence the implementation of the Cachet 3.x design, crafted by [Jump24](https://jump24.co.uk/).

I'd really love help to get the design implemented into the core. In particular, this project is aimed at creating the components and layouts required to make the status page and dashboard.

You won't need to implement every single dashboard page and feature or worry about things like authentication.

We expect to have the designs completed this week. Cachet will retain its overall layout and feel, but with some additions that'll make it feel nicer and more 2023!

The designs include light and dark modes, are fully responsive and have been designed to work with customizable palettes that'll be configurable from the dashboard.

We'll be using Tailwind across Cachet. The status page itself will use Livewire 3 and the dashboard will use Vue + Inertia.

If you can help out with this work, please let me know below!

**Comment by mdavis1982** [comment]
@jbrooksuk I'm definitely up for helping to implement these designs 😄

**Comment by joelbutcher** [comment]
@jbrooksuk super happy to carve out some spare time to dedicate to this project with whatever needs doing, not just on the API side. HMU on twitter (@joelbutcher97) and let me know what you need from me and let's get this baby off the ground 🚀😎🧑🏻‍💻

**Comment by stephenjude** [comment]
I would love to join in implementing this 💯

**Comment by aryehraber** [comment]
Looks like there's already a few people interested, but if you need anyone else I'd be happy to contribute 👍

**Comment by jbrooksuk** [comment]
The design is now completed and ready to be implemented. Yesterday, I implemented a "kitchen sink" route that is accessible at `/status/dashboard/kitchen-sink` that allows us to test and view components.

Now, I need some volunteers to help implement the Vue.js dashboard components and the status page (probably Livewire?) components.

If you're interested, please email me at james[at]cachethq[dot]io and we'll get moving!
